docs/src/main/sphinx/release/release-411.md

```diff
@@ -33,27 +59,80 @@
   with the `openx_json_native_reader_enabled` and
   `openx_json_native_writer_enabled` session properties or the
   `openx-json.native-reader.enabled` and `openx-json.native-writer.enabled`
-  configuration properties. ({16073}``)
+  configuration properties. ({issue}`16073`)
+* Add support for coercion between `char` types of different lengths. ({issue}`16402`)
 * Improve performance of queries with joins where both sides of a join have keys
   with the same table bucketing definition. ({issue}`16381`)
 * Improve query planning performance for queries scanning tables with a large
   number of columns. ({issue}`16203`)
+* Improve scan performance for `COUNT(*)` queries on row-oriented formats. ({issue}`16595`)
+* Ensure the value of the `hive.metastore-stats-cache-ttl` configuration
+  property always is greater than or equal to the value specified in the
+  `hive.metastore-cache-ttl` configuration property. ({issue}`16625`)
+* Skip listing Glue metastore tables with invalid column types. ({issue}`16677`)
 * Fix query failure when a file that is using a text file format with a
   single header row is split. ({issue}`16492`)
-* Fix query failure when reading text files that have been replaced during the
-  query. ({issue}`16510`)
+* Fix potential query failure with a Kerberized Hive connector when the query
```

Comment:
There's no such a thing as "Kerberized Hive connector". Rephrase to "... when Kerberos is enabled and the query execution takes longer than..."